test/jdk/java/nio/Buffer/EqualsCompareTest.java

```diff
@@ -713,4 +722,19 @@ static int[] ranges(int from, int to) {
                         .distinct().toArray();
         }
     }
+
+    @Test
+    void testHashCode() throws IOException {
+        byte[] bytes = "hello world".getBytes(UTF_8);
+        Path path = Files.createTempFile("", "");
+        Files.write(path, bytes);
+        try (FileChannel fc = FileChannel.open(path, StandardOpenOption.READ,
+            StandardOpenOption.DELETE_ON_CLOSE)) {
```

Comment:
If you statically import StandardOpenOption then it might be more readable.

Reply:
Fixed in ba25d55.